applications/ssotap/secrets-idfdev.yaml

```diff
@@ -0,0 +1,11 @@
+"google_creds.json":
+  description: >-
+    Google service account credentials used to write async job output to
+    Google Cloud Storage.
+pgpassword:
+  description: >-
+    Password to external PostgreSQL server that contains the Solar System
+    Objects data.
```

Comment:
`secrets.yaml` and `secrets-<environment>.yaml` files are merged, so you don't need to duplicate the values from `secrets.yaml` here. You only have to add the new one.

Reply:
Ok I see, I've modified this accordingly